common/config/azure-pipelines/integration-pr-validation.yaml

```diff
@@ -23,27 +23,27 @@ variables:
   - group: iModel.js Integration Test Users
 
 jobs:
-  - job: Node_12_x
+  - job: Node_14_x
     condition: succeeded()
     pool:
       vmImage: ubuntu-latest
     steps:
       - template: templates/integration-test-steps.yaml
         parameters:
-          Node_Version: 12.x
-  - job: Node_10_x
+          Node_Version: 14.x
+  - job: Node_12_x
     condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
     pool:
       vmImage: ubuntu-latest
     steps:
       - template: templates/integration-test-steps.yaml
         parameters:
-          Node_Version: 10.x
-  - job: Node_10_17
+          Node_Version: 12.x
+  - job: Node_12_17
```

Comment:
If we stop testing node 10, can we really claim it's still supported?

Reply:
I've been going back and forth on how to still test it and not downgrade the pnpm version because I still want to support Node 10.

Reply:
Node 10 is out of maintenance LTS according to nodes website so will no longer receive updates.

Reply:
sounds good. I'll document that it should still work but that we're no longer actively testing it and recommend moving to the LTS version.

Reply:
I don't see any changes in docs to say 10 should still work but is no longer being actively tested.  Will that be done independently?

Reply:
Just pushed an update to include a Node.js supported version section, https://github.com/imodeljs/imodeljs/pull/1590/files#diff-833d642c80c6b03fb21b71aed050f9cccca35fb45e2481c9f56856d4071632baR15-R20.

Let me know what you think